frontend-thea-aa/src/routes/detailedContaminate/+page.svelte

```diff
@@ -0,0 +1,97 @@
+<script lang="ts">
+    import QualityIndicator from '$lib/components/QualityIndicator.svelte'
+	import Statistic from '$lib/components/Statistic.svelte'
+    import type { ContaminatedSite, DetailedContaminatedSite } from '$lib/types';
+    import { onMount } from 'svelte'
+    import {ChevronLeft, ChevronRight} from "lucide-svelte/icons"
+	import NearAq from '$lib/components/NearAq.svelte'
+	import DetailedContamTable from '$lib/components/DetailedContamTable.svelte'
+
+
+    let { data } = $props()
+
+    let mapElement: HTMLDivElement | null = $state(null)
+    onMount(async () => {
+		const L = (await import('leaflet')).default
+		if (mapElement) {
+			const map = L.map(mapElement).setView([data.location.lat, data.location.long], 13) // change to use props later
+			L.tileLayer('https://{s}.basemaps.cartocdn.com/rastertiles/voyager/{z}/{x}/{y}{r}.png', {
+				attribution: `&copy;<a href="https://www.openstreetmap.org/copyright" target="_blank">OpenStreetMap</a>,
+          &copy;<a href="https://carto.com/attributions" target="_blank">CARTO</a>`,
+				subdomains: 'abcd',
+				maxZoom: 19
+			}).addTo(map)
+		}
+	})
+</script>
+
+<link
+	rel="stylesheet"
+	href="https://unpkg.com/leaflet@1.9.4/dist/leaflet.css"
+	integrity="sha256-p4NxAoJBhIIN+hmNHrzRCf9tD/miZyoHS5obTRR9BMY="
+	crossorigin=""
+/>
+
+<div class="ml-20 mt-5">
+    <div class= "flex-col">
+        <a href="/">
+            <div class="flex flex-row items-center">
+                <ChevronLeft color="#9ca3af" />
+                <p class="text-lg text-gray-400">CONTAMINATED SITES</p>
+            </div>
+        </a>
+        <p class="text-6xl"><b>{data.name}</b></p> 
```

Comment:
<b> tag is bad, never use it. My thought is that this should probably be an h1 tag with "text-6xl font-bold" as the classes